Documentation/gettingstarted/cni-chaining-aws-cni.rst

```diff
@@ -24,7 +24,12 @@ and provide encryption.
 
    Please ensure that you are running version `1.7.9 <https://github.com/aws/amazon-vpc-cni-k8s/releases/tag/v1.7.9>`_
    or newer of the AWS VPC CNI plugin to guarantee compatibility with Cilium.
-   The official upgrade instructions can be found `here <https://docs.aws.amazon.com/eks/latest/userguide/cni-upgrades.html>`_.
+
+   .. code-block:: bash
+
+      $ kubectl -n kube-system get ds/aws-node -o json | jq -r '.spec.template.spec.containers[0].image'
+      602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.7.5-eksbuild.1
```

Comment:
Why is this step necessary?

Reply:
From commit log:

> This commit extends the AWS-CNI documentation with commands to check the
current aws-node version and update it.

I understand that it's not necessary per se, it's used to show how to check what version is running. Could maybe be split into two blocks? “Check version with...” and “If necessary, update...” (Or personally, I'm also good with the current version).

Reply:
Because I don't think you should update if you already have v1.7.9 or newer.

Reply:
Maybe it would be helpful to communicate that?